src/main/java/com/erii/ui/ControlPanel.java

```diff
@@ -0,0 +1,153 @@
+package com.erii.ui;
+
+import java.util.Scanner;
+import com.erii.user.UserDetails;
+
+
+public class ControlPanel {
+
+    public static void main(String[] args) {
```

Comment:
this method is really long (>30 LOC) shorten it by abstracting out parts of code into functions 

https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods